.gitignore

```diff
@@ -13,3 +13,5 @@
 *.swp
 *~
 default.etcd
+testdata/softhsm.conf
+softhsm-slot0.db
```

Comment:
Could you add a comment saying that these are created by the integration tests please?